### PR TITLE
Update vote helpers based off from the new behaviour with aria-pressed

### PR DESCRIPTION
**Pull request by luis-almeida**

### CODE REVIEW

File: templates/article_page.hbs [hunk 1]
```diff
@@ -135,9 +135,9 @@
         {{#with article}}
           <div class="article-votes">
             <span class="article-votes-question">{{t 'was_this_article_helpful'}}</span>
-            <div class="article-votes-controls" role='radiogroup'>
-              {{vote 'up' role='radio' class='button article-vote article-vote-up'}}
-              {{vote 'down' role='radio' class='button article-vote article-vote-down'}}
+            <div class="article-votes-controls">
```
Comment (VilhelmNielsen) [inline templates/article_page.hbs]:
```suggestion
            <span class="article-votes-question" id="article-votes-label">{{t 'was_this_article_helpful'}}</span>
            <div class="article-votes-controls" role="group" aria-labeledby="article-votes-label">
```

File: templates/article_page.hbs [hunk 2]
```diff
@@ -237,7 +237,7 @@
                     </div>
 
                     <div class="comment-actions-container">
-                      <div class="comment-vote vote" role='radiogroup'>
+                      <div class="comment-vote vote">
```
Comment (VilhelmNielsen) [inline templates/article_page.hbs]:
```suggestion
                      <div class="comment-vote vote" role="group">
```

File: templates/community_post_page.hbs [hunk 3]
```diff
@@ -91,7 +91,7 @@
           </div>
 
           <div class="post-actions-wrapper">
-            <div class="post-vote vote" role='radiogroup'>
+            <div class="post-vote vote">
```
Comment (VilhelmNielsen) [inline templates/community_post_page.hbs]:
```suggestion
            <div class="post-vote vote" role="group">
```

File: templates/community_post_page.hbs [hunk 4]
```diff
@@ -206,7 +206,7 @@
 
               <div class="comment-actions-container">
                 {{#unless official}}
-                  <div class="comment-vote vote" role='radiogroup'>
+                  <div class="comment-vote vote">
```
Comment (VilhelmNielsen) [inline templates/community_post_page.hbs]:
```suggestion
                  <div class="comment-vote vote" role="group">
```

**Comment by luis-almeida** [comment]
@VilhelmNielsen updated the PR following your comments. Can give it another look?

**Pull request merged**